sdk/cosmos/src/requests/create_document_builder.json

```diff
@@ -5,22 +5,30 @@
 		"crate::prelude::*",
 		"crate::ResourceType",
 		"crate::responses::CreateDocumentResponse",
-		"azure_sdk_core::errors::{extract_status_headers_and_body, AzureError, UnexpectedHTTPResult}",
-		"azure_sdk_core::prelude::*",
-		"azure_sdk_core::{No, ToAssign, Yes}",
+		"azure_core::errors::{extract_status_headers_and_body, AzureError, UnexpectedHTTPResult}",
+		"azure_core::prelude::*",
+		"azure_core::{No, ToAssign, Yes}",
 		"chrono::{DateTime, Utc}",
 		"hyper::StatusCode",
 		"serde::Serialize",
 		"std::convert::TryFrom",
 		"std::marker::PhantomData"
 	],
 	"inline": true,
-	"extra_types": [ "'a", "'b", "C", "D" ],
-	"extra_wheres": [ "C: CosmosClient", "D: DatabaseClient<C>" ],
+	"extra_types": [
+		"'a",
+		"'b",
+		"C",
+		"D"
+	],
+	"extra_wheres": [
+		"C: CosmosClient",
+		"D: DatabaseClient<C>"
+	],
```

Comment:
What did you use to format the json? We should probably define a standard to avoid polluting the git history with different formats.

Reply:
I believe this was my VS Code JSON extension. I can revert these changes.